pkgbot/__init__.py

```diff
@@ -132,7 +132,7 @@ def process_request(data):
             "{1}".format(repo, fifo_socket_location)
         )
         return
-    fifo = os.open(fifo_socket_location, os.O_NONBLOCK | os.O_WRONLY, buffering=0)
```

Comment:
Was there a reason for this `buffering=0`? Note that I simply removed it because it's not a supported argument for Python3's version of `os.open()`, to "just make it werk", but this might potentially break something else down the line.

Reply:
OK, thanks!